docs/apis/plugintypes/qbank/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
As this page is incomplete, is there any issue to add the missing information?  For now, I think it's OK to leave it as it is, but it would be great to mark it as "WIP" to add more information at any point in the near future :-)

Reply:
This is the current state of the page in the current dev docs.

We currently dont' have anything to do that, but I would like to add something to the frontmatter, like `draft: true` which we then display as a banner and ask for help in fixing.

Reply:
I've raised #83 to do something like this.

I'm happy to remove this doc from this PR for now instead.

Reply:
As you've raised #83, I think it's OK to leave it as it is now and then add the `draft` setting, tag or both.